.github/workflows/unit-testing-and-coverage.yml

```diff
@@ -50,6 +50,9 @@ jobs:
       - name: "Static Type Checking"
         # language=bash
         run: |
+          # Required by examples
+          pip install dnspython argon2-cffi
```

Comment:
pytype checks _all_ .py files. The authrelay example depends on dnspython and argon2-cffi, so I had to install them to make pytype not raise import-error